### ISSUE TITLE
Bug: Meter not working at all with new plugin via BLE

### ISSUE THREAD

**Issue by SeanHama** [OpenWonderLabs/homebridge-switchbot#110]
### Describe The Bug

Meter Device not found - not working in HB

### To Reproduce

Enable BLE, add meter device, restart plugin in HB

### Expected behavior

It should work

### Relevant log output

```shell
I've switched from old switchbot BLE plugin to this, followed all steps and when I enabled the device debug it has the following error. (see underscreenshots)
```

### Config for homebridge-switchbot

```shell
{
            "name": "SwitchBot",
            "credentials": {
                "openToken": "8d",
                "notice": "Keep your token a secret!"
            },
            "options": {
                "devices": [
                    {
                        "deviceId": "F2D21030xxxx",
                        "configDeviceType": "Meter",
                        "ble": true
                    }
                ],
                "debug": "device"
            },
            "_bridge": {
                "username": "0E:CD:CA:9D:A4:F4",
                "port": 35673
            },
            "platform": "SwitchBot"
        }
```

### Screenshots

[25/11/2021, 12:04:31] Registering platform '@switchbot/homebridge-switchbot.SwitchBot'
[25/11/2021, 12:04:31] [SwitchBot] Loaded @switchbot/homebridge-switchbot v1.2.1-beta.6 child bridge successfully
[25/11/2021, 12:04:31] Loaded 0 cached accessories from cachedAccessories.0ECDCA9DA4F4.
[25/11/2021, 12:04:31] [SwitchBot] Using Default Refresh Rate (2 minutes).
[25/11/2021, 12:04:31] [SwitchBot] Using Default Push Rate.
[25/11/2021, 12:04:31] Homebridge v1.3.8 (HAP v0.9.7) (SwitchBot) is running on port 35673.
[25/11/2021, 12:04:32] [SwitchBot] [DEVICE] deviceListInfoStatus: {"statusCode":100,"body":{"deviceList":[{"deviceId":"F2D210301B59","deviceName":"Meter 59","deviceType":"Meter","enableCloudService":false,"hubDeviceId":"000000000000"}],"infraredRemoteList":[]},"message":"success"}
[25/11/2021, 12:04:32] [SwitchBot] Total SwitchBot Devices Found: 1
[25/11/2021, 12:04:32] [SwitchBot] [DEVICE] Device: Meter 59 Neither OpenAPI and BLE Enabled
[25/11/2021, 12:04:32] [SwitchBot] [DEVICE] Unable to Register new device: Meter 59 Meter - F2D210301B59

### Device & Model

Meter

### Node.js Version

v14.18.1

### NPM Version

v6.14.15

### Homebridge Version

latest

### Homebridge Switchbot Plugin Version

1.2.0

### Operating System

Raspbian

**Comment by donavanbecker** [comment]
Do you have a hub? If so does this work without BLE enabled?

**Comment by SeanHama** [comment]
No hub, direct BLE to HB via Rpi4

**Comment by donavanbecker** [comment]
Just released v1.2.1 with a fix for this.

**Comment by SeanHama** [comment]
Awesome I'll test it now
